src/applications/vaos/utils/constants.js

```diff
@@ -215,3 +209,5 @@ export const DISTANCES = [
 export const DISABLED_LIMIT_VALUE = 0;
 export const PRIMARY_CARE = '323';
 export const DIRECT_SCHEDULE_TYPES = new Set([PRIMARY_CARE, '502']);
+export const VA_FACILITY_TYPE = 'vamc';
+export const COMMUNITY_CARE_FACILITY_TYPE = 'communityCare';
```

Comment:
LGTM.  Minor thing but might be a little more consistent with the other constants to group the facility types in a single object like so:

```
export const FACILITY_TYPES = {
  VA: 'vamc',
  COMMUNITY_CARE: 'communityCare'
};